exporters/geneva/example/example_metrics.cc

```diff
@@ -0,0 +1,133 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef ENABLE_METRICS_PREVIEW
```

Comment:
Why put `_PREVIEW` int he macro name? It doesn't sound like this is based on preview metrics implementation.

Reply:
Yeah, the macro actually has no effect, have removed it. Thanks.